sections/earn/LiquidationTab/LiquidationTab.tsx

```diff
@@ -66,16 +66,19 @@ import Connector from 'containers/Connector';
 
 type LiquidationTabProps = {
 	liquidationRewards: Wei;
+	refetchAllRewards: () => void;
```

Comment:
Yeah there is some prop drilling here, not ideal and would like to abstract away, but I think it would be better to re-write when we do the refresh/refactor.